### ISSUE TITLE
Decrypt Functionality

### ISSUE THREAD

**Issue by justincpresley** [ryangriggs/PDFEncrypt#3]
It would be really convenient to have a decrypt option - remove encryption given a password and a source file.

This application could have two tabs "Encrypt" and "Decrypt" with the Encrypt tab being highlighted in some way and shown first. 
You could also move Settings up to be a tab as well (but I do like the additional popup).

**Comment by ryangriggs** [comment]
I agree and this feature will be added to V2.  Thanks

**Comment by cachius** [comment]
Good idea. iText is included anyway, and readily [supports PDF decryption](https://kb.itextpdf.com/itext/encrypting-decrypting-pdfs).

**Comment by ryangriggs** [comment]
Yes this is a planned feature for V2.
